content/en/docs/contribute/start.md

```diff
@@ -243,7 +243,11 @@ your fork's repository. The GitHub website will prompt you to create the
 pull request if it detects that you pushed a new branch to your fork.
 {{< /note >}}
     
-5.  The **Open a pull request** screen appears. The subject of the pull request
+5.  {{< note >}}
```

Comment:
@MaxKam Thank you for submitting this!  Although your reasoning is excellent, putting the note where you did makes the bullet point look out of place compared to the other bullet points with notes. Please see the rendered version [here](https://deploy-preview-15773--kubernetes-io-master-staging.netlify.com/docs/contribute/start/#review-docs-pull-requests). 
Would it be possible for you to place the note at the bottom of the bullet point content? Thanks!

Reply:
@aimeeu fixed in 230b54c.